### DISCUSSION TITLE
Make users navigate to their own resource or profile page

### DISCUSSION THREAD

**Discussion by classicalguss** [filamentphp/filament#12848] [Help]
### Package

Panel builder

### Package Version

v3.0.0

### How can we help you?

I have a CompanyResource that lists, edit, delete, and view companies intended for Admin Users. However I do have company customers that also have their own admin panel.

I want them to have a page were they can view their own company, but not show them a list of other companies. How is it possible to do in Filament? I tried to check Pages, Widgets, Navigation. Nothing seems to work perfectly. I had some success with NavigationItem, but unfortunately I have to register the resource, and the index link will show up in the breadcrumbs.

**Comment by dissto** [comment]
Since you mention that "company customers" have their own panel. Couldn't you just "scope" the query like

```php
$table
  ->modifyQueryUsing(fn (Builder $query) => $query->whereBelongsTo(auth()->user()))
```

Untested and just a guess. Perhaps you could share some code, then it would be easier to give a tailored suggestion.

**Reply by classicalguss** [reply]
Thank you for the reply. But no this is not what I want. I want them to have a direct link in the sidebar to the details page. Where they can edit their company profile. So the page I'm looking for wouldn't be a List page, it would be a View or Edit page.

**Comment by classicalguss** [comment]
I figured it out. You have to create a normal page `php artisan make:filament-page EditCompanyProfile`, and put the filament form in the livewire view page by following this documentation https://filamentphp.com/docs/3.x/forms/adding-a-form-to-a-livewire-component
